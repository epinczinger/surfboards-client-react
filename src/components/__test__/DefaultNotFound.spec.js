import React from 'react';
import renderer from 'react-test-renderer';
import DefaultNotFound from '../DefaultNotFound';

describe('The DefaultNotFound component', () => {
  it('renders as expected matching snapshot', () => {
    const tree = renderer.create(<DefaultNotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
