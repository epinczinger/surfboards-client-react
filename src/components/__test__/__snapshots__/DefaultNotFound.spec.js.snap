// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The DefaultNotFound component renders as expected matching snapshot 1`] = `
<h1
  className="mx-auto my-auto text-center text-3xl text-red-500 font-bold"
>
  Error, this page doesn't exist.
  <br />
   
  Try another one!.
</h1>
`;
